Journal/Journal/Model/EntryRepresentation.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  EntryRepresentation.swift
+//  Journal
+//
+//  Created by Bradley Yin on 8/21/19.
+//  Copyright © 2019 bradleyyin. All rights reserved.
+//
+
+import Foundation
+
+struct EntryRepresentation: Codable, Equatable {
+    var mood: Int64?
+    var bodyText: String?
+    var identifier: UUID?
+    var title: String?
+    var timeStamp: Date?
+}
+
+func == (lhs: EntryRepresentation, rhs: Entry) -> Bool {
+    guard let identifier1 = lhs.identifier, let identifier2 = rhs.identifier else { return false }
```

Comment:
There's no need to unwrap the identifiers. In fact, doing so could result in a false negative match.

the next line is completely fine, but is more maintainable if you did

```
    return identifier1 == identifier2 &&
		lhs.title == rhs.title &&
		lhs.mood == rhs.mood &&
		rhs.bodyText == lhs.bodyText &&
		rhs.timeStamp == lhs.timeStamp
```